src/actions/channels.js

```diff
@@ -827,17 +843,34 @@ export function markChannelAsRead(channelId, prevChannelId) {
             }
         }
 
+        if (teamMember) {
+            actions.push({
+                type: TeamTypes.RECEIVED_MY_TEAM_UNREADS,
+                data: [teamMember]
+            });
+        }
+
         if (actions.length) {
             dispatch(batchActions([...actions]), getState);
         }
     };
 }
 
-export function markChannelAsUnread(channelId, mentionsArray) {
+export function markChannelAsUnread(teamId, channelId, mentionsArray) {
```

Comment:
Is it possible to get the team id from the channel object so we don't need to pass it in here? If it's too much work then this is fine, 0/5